src/ripple/rpc/impl/ServerHandlerImp.cpp

```diff
@@ -247,9 +247,9 @@ build_map(boost::beast::http::fields const& h)
         auto key (e.name_string().to_string());
         // TODO Replace with safe C++14 version
```

Comment:
(Not related to this PR), but what does this comment mean exactly? I remember a PR when @HowardHinnant explained the gotchas with the argument to `tolower` (requiring the cast to `unsigned char`), but I don't understand the `TODO` comment.

Reply:
I don't really understand this comment either.  Nor the use of `::tolower`.  We don't explicitly `#include <ctype.h>` (where `::tolower` lives), and we don't normally call the C versions of functions.

I recommend removing the comment and calling `std::tolower`.  There's another `::tolower` at src/ripple/beast/core/LexicalCaast.h:195.

Reply:
I don't understand the comment.I think I'm going to leave this issue alone in this PR. We can change it in another PR.